Microsoft.Toolkit.Uwp.SampleApp/SamplePages/MasterDetailsView/MasterDetailsView.bind

```diff
@@ -71,6 +71,26 @@
                     </StackPanel>
                 </DataTemplate>
             </controls:MasterDetailsView.NoSelectionContentTemplate>
+            <controls:MasterDetailsView.MasterCommandBar>
+                <CommandBar>
+                    <AppBarButton Icon="Back" Label="Back"/>
```

Comment:
Could be useful to have a local command which just causes a message box pop-up to show how to hook these buttons up to something.  Similar to how the [DismissCommand](https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/InAppNotification) was hooked in to the sample for the InAppNotification?

Reply:
I'm a fan of keeping things simple and only include code that is relevant to the control. :)

Reply:
@michael-hawker I think the [CommandBar documentation](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.CommandBar) does a good job of explaining how the CommandBar works. I was thinking about wiring up a viewmodel and actually have the previous/next buttons wired into that but thought that wasn't really needed.

Reply:
Sounds good, could be a future doc improvement then.